modules/openapi-generator/src/main/resources/python-pydantic-v1/api_client.mustache

```diff
@@ -567,10 +567,10 @@ class ApiClient:
         if collection_formats is None:
             collection_formats = {}
         for k, v in params.items() if isinstance(params, dict) else params:  # noqa: E501
-            if isinstance(v, (int, float)):
-                v = str(v)
             if isinstance(v, bool):
                 v = str(v).lower()
+            if isinstance(v, (int, float)):
```

Comment:
In python boolean is a subclass of int so `isinstance(True, (int, float)) == True`. We need to check if the type is boolean before checking if the type is a number to get the behaviour we desire.

Reply:
> In python boolean is a subclass of int so `isinstance(True, (int, float)) == True`. We need to check if the type is boolean before checking if the type is a number to get the behaviour we desire.

TIL it's [still the case in Python 3](https://docs.python.org/3/library/stdtypes.html#boolean-type-bool), I thought this was only the case in 2.x 👍